doc/contribute/documentation/guidelines.rst

```diff
@@ -30,16 +30,40 @@ This document provides a quick reference for commonly used reST and
 Sphinx-defined directives and roles used to create the documentation
 you're reading.
 
+Content Structure
+*****************
+
+Tabs, spaces, and indenting
+===========================
+
+Indenting is significant in reST file content, and using spaces is
+preferred.  Extra indenting can (unintentionally) change the way content
+is rendered too.  For lists and directives, indent the content text to
+the first non-white space in the preceding line.  For example::
+
+   * List item that spans multiple lines of text
+     showing where to indent the continuation line.
+
+   1. And for numbered list items, the continuation
+      line should align with the text of the line above.
+
+   .. code-block::
+
+      The text within a directive block should align with the
+      first character of the directive name.
+
+Refer to the Zephyr :ref:`coding_style` for additional requirements.
+
 Headings
-********
+========
 
-While reST allows use of both and overline and matching underline to
+While reST allows use of both an overline and matching underline to
 indicate a heading, we only use an underline indicator for headings.
 
-* Document title (h1) use "#" for the underline character
-* First section heading level (h2) use "*"
-* Second section heading level (h3) use "="
-* Third section heading level (h4) use "-"
+* Document title (h1) use ``#`` for the underline character
+* First section heading level (h2) use ``*``
+* Second section heading level (h3) use ``=``
+* Third section heading level (h4) use ``-``
 
 The heading underline must be at least as long as the title it's under.
```

Comment:
We usually make them exactly as long as the title they are under. Should make that the guideline?

Reply:
Ya, agreed. I didn't change the original wording from https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#headings but this too caught my eye. I see no reason why we wouldn't make equal length a guideline indeed.

Reply:
addressed. Thanks!